Formula/txt2tags.rb

```diff
@@ -1,9 +1,8 @@
 class Txt2tags < Formula
   desc "Conversion tool to generating several file formats"
   homepage "https://txt2tags.org/"
-  url "https://github.com/txt2tags/txt2tags/archive/2.6.tar.gz"
```

Comment:
This has been written in python
https://github.com/txt2tags/txt2tags/tree/v3 (py source code) vs
https://github.com/txt2tags/txt2tags/tree/v2 (binary install)